ble/services/DFUService.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __BLE_DFU_SERVICE_H__
 #define __BLE_DFU_SERVICE_H__
 
+#ifdef TARGET_NRF51822 /* DFU only supported on nrf51 platforms */
```

Comment:
@andresag01 
 #ifndef __XXX_H__ is a convention for guarding multiple inclusions, hence I would rather not touch that. From working in MPG, it is rare to see complicated #if statements. I prefer doing one thing on one line to keep it clear.